### PR TITLE
Unused codes

### PR DESCRIPTION
**Pull request by favadi**

### CODE REVIEW

**Comment by favadi** [comment]
[Cleanup unused codes](https://trello.com/c/C1OfOYss/51-cleanup-unused-codes)

File: common/bolt.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package common
```
Comment (tranvictor) [inline common/bolt.go]: I dont think putting bolt specific utils in common is appropriate. In my initial ideal, common is a package that defines the types and utils to be shared among highest level components (core, data, stat).
To those packages, they always see abstracted storage with specific interfaces, not bolt.
**Proposal:** I think an util file inside of bolt specific package makes more sense.

Comment (favadi) [reply common/bolt.go]: I did that in 45481ab.

File: cmd/clicmd/startserver.go [hunk 2]
```diff
@@ -24,49 +20,12 @@ const (
...
 var servPort int = 8000
 var endpointOW string
-var base_url, auth_url string
+var base_url string
 var enableStat bool
 var noCore bool
 var stdoutLog bool
 var dryrun bool
 
-func backupLog(arch archive.Archive) {
```
Comment (cyberliem) [inline cmd/clicmd/startserver.go]: This function is necessary. Don't delete it. The function call has been accidentally  deleted

Comment (favadi) [reply cmd/clicmd/startserver.go]: @cyberliem so.. the logging backup feature has been broken for a while? Could you push a commit to revert my changes to this function and add the call back?

Comment (cyberliem) [reply cmd/clicmd/startserver.go]: I update my latest commit

Comment (tranvictor) [reply cmd/clicmd/startserver.go]: @favadi it is not deployed to production nor staging yet so don't worry.

**Pull request merged**